package.json

```diff
@@ -145,7 +145,6 @@
     "react-test-renderer": "~16.14.0",
     "shadow-cljs": "2.11.20",
     "style-loader": "^0.19.0",
-    "uglifyjs-webpack-plugin": "^1.0.0",
```

Comment:
# Remove `uglifyjs-webpack-plugin`

```
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
```

Related reading material:
- https://webpack.js.org/plugins/terser-webpack-plugin
> If you are using webpack v5 or above you do not need to install this plugin. Webpack v5 comes with the latest terser-webpack-plugin out of the box. 

- https://github.com/webpack-contrib/terser-webpack-plugin/issues/15
- https://github.com/webpack/webpack/issues/7923
- https://stackoverflow.com/questions/65298689/uglifyjs-and-webpack-v5